packages/experience-builder-sdk/src/blocks/VisualEditor.tsx

```diff
@@ -1,9 +1,27 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import { InternalSDKMode } from '../types';
-import { componentRegistry } from '../core/componentRegistry';
-import VisualEditor from '@contentful/experience-builder-visual-editor';
 import { VisualEditorContextProvider } from './editor/VisualEditorContext';
 
+const version = '0.0.1-pre-20231212T210325.0';
+const scriptUrl = `https://unpkg.com/@contentful/experience-builder-visual-editor@${version}/dist/renderApp.js`;
+
+const VisualEditor: React.FC = () => {
+  useEffect(() => {
+    const script = document.createElement('script');
+    script.type = 'module';
+    script.src = scriptUrl;
+    script.async = true;
+
+    document.body.appendChild(script);
+
+    return () => {
+      document.body.removeChild(script);
+    };
+  }, []);
+
+  return <div id="cf-visual-editor" />;
```

Comment:
Can we move this id value `cf-visual-editor` to a constant in the `core` package so that both the visual-editor and sdk can import that constant instead?

Reply:
That's a great idea. I'll work on this change